kubernetes/hmda-dashboard/templates/deployment.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "hmda-dashboard.fullname" . }}
+  labels:
+    app: {{ template "hmda-dashboard.name" . }}
+    chart: {{ template "hmda-dashboard.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "hmda-dashboard.name" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "hmda-dashboard.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      volumes:
+        - name: tz-config
+          hostPath:
+            path: /usr/share/zoneinfo/America/New_York
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          volumeMounts:
+            - name: tz-config
+              mountPath: /etc/localtime
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          resources:
+            limits:
```

Comment:
This should also have a `requests` section or the values from values.yaml should be leveraged. hmda-platform deployment.yaml does this